### DISCUSSION TITLE
Problem with installation of a demo environment

### DISCUSSION THREAD

**Discussion by DavidPlato90** [pimcore/pimcore#17432] [Q&A / Support]
Hello, I am trying to set up a demo environment following the installation guide on the official site: https://pimcore.com/docs/platform/Pimcore/Getting_Started/Installation/Webserver_Installation/
But pity that I met a problem while using Composer, and could not finish the installation. Is there any mistake, or is any required preparation incomplete, or is the installation charged with the official guide?
![Capture](https://github.com/user-attachments/assets/3c3ce6af-65e3-4eaa-99f5-ec0640b44c31)

**Comment by kingjia90** [comment]
It looks like it's not installing the latest released packages

Could you please try to `composer clear-cache` before creating project?

As seen on https://laracasts.com/discuss/channels/general-discussion/composer-not-installing-latest-version

**Reply by DavidPlato90** [reply]
Thanks, It works better now.
But I met a new problem, it seems that the password of root of MySQL is incorrect, but I haven't seen anywhere to input this while installation, and there is no explain about what password the MySQL root should be set.
The error message while composer is: #message: "An exception occurred in the driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)"

**Reply by blankse** [reply]
There is an open bug https://github.com/pimcore/demo/issues/591
